cmd/containerd/command/main_unix.go

```diff
@@ -61,11 +61,10 @@ func handleSignals(ctx context.Context, signals chan os.Signal, serverC chan *se
 						log.G(ctx).WithError(err).Error("notify stopping failed")
 					}
 
-					if server == nil {
-						close(done)
-						return
+					if server != nil {
+						server.Stop()
```

Comment:
I cant remember, is `server.Stop` a blocking call?  If so, should we move the cancel before this call?

Reply:
Yeah I can block.
Went ahead and moved the cancel.